packages/cloud/src/libraries/tenants.ts

```diff
@@ -67,6 +68,12 @@ export class TenantsLibrary {
     });
   }
 
+  async updateTenantById(tenantId: string, payload: PatchTenant): Promise<TenantInfo> {
+    const { id, name, tag } = await this.queries.tenants.updateTenantById(tenantId, payload);
+
+    return { id, name, tag, indicator: getManagementApiResourceIndicator(id) };
+  }
```

Comment:
nit:

```js
async updateTenantById(tenantId: string, payload: PatchTenant): Promise<TenantInfo> => 
({
  ...await this.queries.tenants.updateTenantById(tenantId, payload)
  indicator: getManagementApiResourceIndicator(id
})
```

Reply:
The returning result of tenant `updateTenantById` query also returns `dbUser` and `dbUserPassword` which can be used to build DB connection. We just pick id, name and tag for security consideration.